crt_portal/cts_forms/templates/forms/complaint_view/show/index.html

```diff
@@ -61,7 +61,7 @@ <h2 class="margin-right-205 margin-top-0 margin-bottom-0 backend-blue">
         {% include 'partials/messages.html' %}
       </div>
     </div>
-    <div class="grid-row grid-gap-lg">
+    <div class="grid-row grid-gap-5">
```

Comment:
Not sure if we wanted to change the gutter size here, but did so for consistency's sake. Easy to undo if we want to keep as-is.